pkg/acs/calacs/include/acs.h

```diff
@@ -13,6 +13,9 @@
 # define ACS_FITS_REC       82
 # define SZ_STRKWVAL        68
 
+#define ACS_WFC_N_COLUMNS_PER_CHIP_INCL_OVERSCAN 4144
```

Comment:
So ``ctePars->columnOffset`` is obatined from [here](https://github.com/jamienoss/hstcal/blob/4e2c39d8ff67007ffd86977ba9e6947d1a017cb3/pkg/acs/calacs/acscte/dopcte-gen2.c#L106-L107)
```c
...
 if (get_amp_array_size_acs_cte(acs, chipImage, ampID, amploc, ccdamp,
                               &amp_xsize, &amp_ysize, &amp_xbeg,
                               &amp_xend, &amp_ybeg, &amp_yend))
        {
            freeOnExit(&ptrReg);
            return (status);
        }
...
ctePars->columnOffset = amp_xbeg;
ctePars->rowOffset = amp_ybeg;
...
```

where ``amp_xbeg`` gives the offset of the image in reference to the beginning of the chip excluding overscan regions (right?). So even if it the subarray went over the chip boundary this would be equivalent to just two individual subarrays on different chips.